### PR TITLE
Add support for Angular 13

### PR DESCRIPTION
**Pull request by Olster**
- Re-generated the lib with new ng version
- Removed karma dependencies (no tests were there)
- Migrated to eslint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
-  "version": "0.2.0",
+  "version": "1.0.0",
   "license": "MIT",
   "repository": "https://github.com/hetznercloud/ngx-translate-mock",
   "scripts": {
     "ng": "ng",
-    "build": "ng build && yarn copy:readme",
-    "lint": "ng lint",
-    "pretty": "prettier --list-different --config ./.prettierrc \"**/*.@(js|ts|scss|md|json|yml|html)\"",
-    "pretty:write": "prettier --write --config ./.prettierrc \"**/*.@(js|ts|scss|md|json|yml|html|template)\"",
```
Comment (wachri) [inline package.json:11]: Seems to be missing?

**Review by wachri — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Olster) [reply package.json:11]: Good catch -- added prettier back in and reformatted the code.

File: package.json [hunk 2]
```diff
@@ -1,40 +1,48 @@
 {
   "name": "@hetznercloud/ngx-translate-mock",
-  "version": "0.2.0",
+  "version": "1.0.0",
   "license": "MIT",
   "repository": "https://github.com/hetznercloud/ngx-translate-mock",
   "scripts": {
     "ng": "ng",
-    "build": "ng build && yarn copy:readme",
```
Comment (wachri) [inline package.json:8]: `&& yarn copy:readme` seems also to be messing here

Comment (Olster) [reply package.json:8]: Added as discussed

**Review by wachri — APPROVED** [review]

**Pull request merged**